poseidon2-air/examples/prove_poseidon2_koala_bear_keccak.rs

```diff
@@ -76,6 +73,9 @@ fn main() -> Result<(), impl Debug> {
         PARTIAL_ROUNDS,
     >(inputs);
 
+    type Dft = Radix2DitParallel;
+    let dft = Dft {};
```

Comment:
You don't want to use the recursive DFT here?

Reply:
Ahh I think I've answered my own question here, seems like it's a little slower.

I wonder if this is due to Radix2DitParallel making use of packed fields.

Reply:
Yeah, the recursive DFT isn't noticeably better yet for this example. Not sure if packed fields is the reason; it might be, though the compiler already generates pretty good AVX2 code for the recursive DFT.